h2o-py/h2o/model/metrics_base.py

```diff
@@ -1428,11 +1428,13 @@ def plot(self, type="roc", server=False, save_plot_path=None, plot=True, **kwarg
         >>> cars_gbm.plot(type="pr")
         
         """
-
+        assert type in ["roc", "pr", "gains_lift"]
```

Comment:
the `plot` method also has unused `kwargs` param!
I don't get it
We allow user to pass whatever without validation…